config/ipp_pilot_usps_facilities.json

```diff
@@ -16,7 +16,6 @@
       "distance": null,
       "streetAddress": "900 E FAYETTE ST RM 118",
       "city": "BALTIMORE",
-      "phone": "410-347-4202",
```

Comment:
I would consider keeping these in the fixtures, because that way they continue to represent information returned by the API, even if the app no longer uses it

Reply:
or wait... is this file not a fixture? it's in `config/`

Reply:
currently this isn't a full representation of what the api returns as it doesn't have the tty field so I feel like it wouldn't be too out of place to remove phone.

Reply:
Hmm, I'm thinking it maybe makes more sense to leave the mocks closer to what the API returns, since this is really a change to the frontend rather than to the API itself. @zachmargolis I can undo this change. @svalexander Do you agree with what I said? Curious to get your feedback.

Reply:
i see your point. i don't feel strongly about it.